From 06d499fb1b54a880ec59700a3bae9fb253310a28 Mon Sep 17 00:00:00 2001
From: Neil Chen <Chen.NeilZX@inventec.com>
Date: Thu, 13 May 2021 03:01:02 +0000
Subject: [PATCH] Starscream - Modify RGMII TX driving strength

    Symtom/Reason:
        RGMII signal can't pass in EV test

    Root Cause:
        Amplitude of TX data isn't enough

    Solution/Change:
        Add MAC# driving strength

    Entry Test:
	sysadmin@transformers:~# devmem 0x1e6e2458
	0x0000000F
---
 drivers/clk/clk-ast2600.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/clk/clk-ast2600.c b/drivers/clk/clk-ast2600.c
index fc98686eaf78..0e71cb6ae79e 100644
--- a/drivers/clk/clk-ast2600.c
+++ b/drivers/clk/clk-ast2600.c
@@ -1122,7 +1122,7 @@ static void __init aspeed_g6_cc_init(struct device_node *np)
 		     ASPEED_G6_DEF_MAC34_DELAY_10M);
 
 	/* MAC3/4 default pad driving strength */
-	regmap_write(map, ASPEED_G6_MAC34_DRIVING_CTRL, 0x0000000a);
+	regmap_write(map, ASPEED_G6_MAC34_DRIVING_CTRL, 0x0000000f);
 	
 	/* RSA clock = HPLL/3 */
 	regmap_update_bits(map, ASPEED_G6_CLK_SELECTION1, BIT(19), BIT(19));	
-- 
2.17.1

